src/ControlSystem/Tags.hpp

```diff
@@ -109,6 +126,63 @@ struct Controller : db::SimpleTag {
   using type = ::Controller<DerivOrder>;
 };
 
+/// \ingroup ControlSystemGroup
+/// The FunctionsOfTime initialized from a DomainCreator, initial time
+/// step, and control system OptionHolders.
+struct FunctionsOfTimeInitialize : domain::Tags::FunctionsOfTime,
```

Comment:
Please move this into a separate file, `ControlSystem/Tags/FunctionsOfTimeInitialize.hpp`

Reply:
For consistency should I also move the whole `Tags.hpp` file to `Tags/Tags.hpp` (while also moving `FunctionsOfTimeInitialize` to a new file)?

Reply:
No I'd leave it where it is for now. The individual tags may eventually migrate to separate files though.